# image coordinate whose origin is the upper left corner of the image
# duckietown_msgs/Pixel uv
duckietown_msgs/Vector2D normalized_uv
---
# ground plane cooridnate whose origin is the center of the robot
geometry_msgs/Point gp
